examples/nextjs/pages/index.js

```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head'
+import styles from '../styles/Home.module.css'
+import Carousel from '../../../src/index'
+
+const colors = [
+  '7732bb',
+  '047cc0',
+  '00884b',
+  'e3bc13',
+  'db7c00',
+  'aa231f',
+  'e3bc13',
+  'db7c00',
+  'aa231f'
+];
+
+export default function Home() {
+
+  const slides = colors.map((color, index) => (
+    <img
+      src={`https://via.placeholder.com/400/${color}/ffffff/&text=slide${
+        index + 1
+      }`}
+      alt={`Slide ${index + 1}`}
+      key={color}
+      style={{
+        height: 400
+      }}
+    />
+  ));
+
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>Nuka Carousel | FormidableLabs</title>
```

Comment:
let's keep it consistent with `Formidable Labs` here and on L36

Reply:
Done, thanks